recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - "10.13"                # [osx and x86_64]
 MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - "10.12"                # [osx and x86_64]
-
+  - "10.13"                # [osx and x86_64]
```

Comment:
I don't see why this is bumped. Error in CI says it needs 10.12+.

Reply:
yeah, I'll put it back, thanks!